src/engine/Eval_generic.ml

```diff
@@ -520,12 +520,13 @@ let bindings_to_env (config : Rule_options.t) ~file bindings =
     bindings
     |> List_.map_filter (fun (mvar, mval) ->
            let try_bind_to_exp e =
+             UCommon.(pr2 (spf "try bind %s" ([%show: AST_generic.expr] e)));
```

Comment:
we should not use pr2 ... we should use Logs.debug here.

Reply:
oops, these were accidentally included (I was debugging). Did you mean to merge with them in?